build/CopyDependencies/CopyDependencies.targets

```diff
@@ -0,0 +1,29 @@
+﻿<Project>
+  <Import Project="..\..\src\EmbeddedSonarAnalyzer.props" />
+
+  <Target Name="Build">
+    <CallTarget Targets="ClearDirs" />
+    <CallTarget Targets="CopyJars" />
+  </Target>
+
+  <PropertyGroup>
+    <JarDownloadDir>$(LOCALAPPDATA)\SLVS_Build_DownloadedJars</JarDownloadDir>
+    <EmbeddedJarDir>..\..\src\Integration.Vsix\DownloadedJars</EmbeddedJarDir>
+  </PropertyGroup>
+  
+  
+  <Target Name="ClearDirs">
+    <RemoveDir Directories="$(EmbeddedJarDir)" />
+  </Target>
+
+  <Target Name="CopyJars">
+    <ItemGroup>
+      <SourceJars Include="$(JarDownloadDir)\sonar-cfamily-plugin-$(EmbeddedSonarCFamilyAnalyzerVersion).jar" />
```

Comment:
Could we somehow centralize the list of dependencies between this and DownloadDependencies task? Currently, if we ever add another analyzer, we would need to modify these 2 targets + EmbeddedSonarAnalyzer.props file

Reply:
I'll add another ticket for this and we can tackle it if we have time in this sprint or we can handle it in a hardening sprint.